osbs/build/build_request.py

```diff
@@ -377,17 +377,34 @@ def adjust_for_registry_api_versions(self):
     def adjust_for_triggers(self):
         """
         Remove trigger-related plugins if no triggers set
+        import_image plugin is only removed for non-custom base images
```

Comment:
So is `import_image` removed for *all* images, or only non-base images? I'm still confused after reading the docstring and the code. :(